home/views.py

```diff
@@ -166,4 +170,15 @@ def get_weather_data(latitude,longitude):
     weather_info['last updated time']=city_weather['current']['last_updated']
     return weather_info
     
-
+def get_top_posts():
+    top_posts = []
+    all_posts = Post.objects.all() 
+    for post in all_posts:
+        if post.likes > 0 or post.dislikes > 0:
+            top_posts.append(post)
+    try:
+        top_posts.sort(key=lambda x: x.likes/x.dislikes, reverse=True)
+    except:
+        top_posts.sort(key=lambda x: x.likes, reverse=True)
```

Comment:
Consider those blogs for "Top Blogs" too where #Likes and #Dislikes are both 0 as the section is empty even though blogs are present.

Reply:
@kunal2812 sir can you check the changes?